lms/static/sass/views/_login-register.scss

```diff
@@ -130,10 +130,11 @@ $sm-btn-linkedin: #0077b5;
     .toggle-form {
         @include box-sizing(border-box);
         max-width: 650px;
-        min-width: 400px;
+        min-width: 250px;
         margin: 0 auto;
         padding-left: $baseline; // Don't want to override any top or bottom (CR)
-        padding-right: $baseline // Don't want to override any top or bottom (CR)
+        padding-right: $baseline; // Don't want to override any top or bottom (CR)
+        padding-bottom: ($baseline/2);
```

Comment:
@AlasdairSwan 
about this new "padding" class:
I am going to apply it on all forms templates where "padding-bottom" issue was seen. Am I right?
can you please clarify this to me?